api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -349,6 +395,16 @@ message Cluster {
     // endpoint metadata if the endpoint metadata matches the value exactly OR it is a list value
     // and any of the elements in the list matches the criteria.
     bool list_as_any = 7;
+
+    // Fallback mechanism that allows to try different route metadata until a host is found.
```

Comment:
One thing that should be made clear (and I'm not sure if that should be here or somewhere else) is how this interacts with the existing LbSubsetFallbackPolicy.

Reply:
I tried to make it clear in API docs:

```
    // If load balancing process, including all its mechanisms (like
    // :ref:`fallback_policy<envoy_v3_api_field_config.cluster.v3.Cluster.LbSubsetConfig.fallback_policy>`)
    // fails to select a host, this policy decides if and how the process is repeated using another metadata.
```
https://github.com/envoyproxy/envoy/pull/22442/files#diff-cd281142432bf3479ab7fbd00d13d37d5f7dbfc47e6724a8f8fdeb1b6f86553eR400

Do you mean to add this note also here, or to generally make it more clear / explain using other words?

Reply:
I think that's fine. Not sure why I missed it before.